util/buildflags/export.go

```diff
@@ -190,8 +190,8 @@ func ParseAnnotations(inp []string) (map[exptypes.AnnotationKey]string, error) {
 			continue
 		}
 
-		typesSplit := strings.Split(types, ",")
-		for _, typeAndPlatform := range typesSplit {
+		typesSplit := strings.SplitSeq(types, ",")
```

Comment:
nit: Not new code but this var definition doesn't really seem to be needed in here and in `docker.go`